src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
```

Comment:
Can be simplified
const children = people.filter(p => (
    people.some(m => m.name === p.mother) && (!onlyWithSon || p.sex === 'm')
  ));